package.json

```diff
@@ -34,6 +34,7 @@
 		]
 	},
 	"dependencies": {
+		"@automattic/calypso-data-stores": "file:./packages/calypso-data-stores",
```

Comment:
Yeah, was considering something like those. Didn't mean to hog such prominent names :sweat_smile: and maybe rename later once the package has proven its value (similar to the `@automattic/calypso-ui` to `@automattic/components` rename.

I think `@automattic/data` sends the wrong message since people might assume is does something similar as `@wordpress/data` (especially since there's precedent with `components` packages in both namespaces, where that's actually true).

`calypso-data-stores` is really pretty ugly though. Maybe I'll go with `data-stores` :thinking: